lvmd/device_class_manager.go

```diff
@@ -138,26 +138,33 @@ func ValidateDeviceClasses(deviceClasses []*DeviceClass) error {
 
 // DeviceClassManager maps between device-classes and volume groups.
 type DeviceClassManager struct {
-	defaultDeviceClass  *DeviceClass
-	deviceClassByName   map[string]*DeviceClass
-	deviceClassByVGName map[string]*DeviceClass
+	defaultDeviceClass    *DeviceClass
+	deviceClassByName     map[string]*DeviceClass
+	deviceClassByVGName   map[string]*DeviceClass
+	deviceClassByPoolName map[string]*DeviceClass
```

Comment:
In this case, pool name always means thin pool name. So, how about rename this field to `deviceClassByThinPoolName`?  In addition, it's nice to add a comment to show a device class is in either deviceClassByVGName or deviceClassByPoolName. It would make the code more descriptive.

Reply:
makes sense, will change the var name and add comments.